modules/tournament2/breaks.php

```diff
@@ -2,6 +2,7 @@
 
 $dsp->NewContent(t('Turnier - Pausenverwaltung'), t('Trage Startzeitpunkt und dauer der Pause ein. Lansuite wird entsprechend spätere Rundenzeiten berechnen'));
 
+$stepParameter = $_GET['step'] ?? 0;
 switch ($_GET['step']) {
```

Comment:
I think the intent to replace the next line with $stepParameter?

Reply:
You are correct. Fixed in https://github.com/lansuite/lansuite/pull/657/commits/71930e5cc5033824b707ff6c7b09aa3c86df0d93